bin/jsonGenerator.js

```diff
@@ -112,46 +39,24 @@ const calculateRootHash = (cipFilePath) => {
 // **********************************************************************************************************
 // **********************************************************************************************************
 
-const generateMetadataJsonFile = (cipJsonFilePath, metadataFilePath, actionType, releaseNumber, releaseName, offChainStoragePath, cipRootHash, secretKey, publicKey) => {
+const generateMetadataJsonFile = (cipJsonFilePath, metadataFilePath, actionType, releaseComment, offChainStoragePath, cipRootHash) => {
```

Comment:
Please align the parameters' names with those in the index.js file

Reply:
Is this what you mean here 79fbe06 or do you want me to add underscore to each of the parameter name?